src/Http/GraphRequest.php

```diff
@@ -431,20 +428,18 @@ protected function getConcatenator()
     * @param string $proxyPort The port to forward
     * requests through. If null, a proxy is not used
     *
-    * @return \GuzzleHttp\Client The new client
+    * @return array
     */
-    protected function createGuzzleClient($proxyPort = null)
+    protected function getGuzzleOptions($proxyPort = null)
```

Comment:
Could this be `getDefaultGuzzleRequestOptions`? Or basically add `default` somewhere in the naming?